dz13/src/app.js

```diff
@@ -0,0 +1,121 @@
+function isSomeProductInStock(productsList) {
+  return productsList.some((product) => product.inStock)
+}
+
+function makeUserTotalBalance(transactions) {
+  const newTransactions = transactions.reduce((result, transaction) => {
+    if (transaction.type === 'income') {
+      return result + transaction.amount
+    }
+    return result - transaction.amount
+  }, 0)
+  return newTransactions
+}
```

Comment:
```suggestion
function makeUserTotalBalance(transactions) {
  return transactions.reduce((result, transaction) => (
    transaction.type === 'income'
      ? result + transaction.amount
      : result - transaction.amount
  ), 0)
}
```

урокоченный вариант с тернарным оператором и без создания лишней переменной

тернарный можно и в одну строчку написать, но мне кажется, что так читается легче